auth_session_timeout/models/res_users.py

```diff
@@ -17,28 +17,70 @@
 class ResUsers(models.Model):
     _inherit = 'res.users'
 
-    def _check_session_validity(self, db, uid, passwd):
+    def _auth_timeout_urls_get(self, session, db, uid):
+        # Pluggable method for calculating ignored urls
```

Comment:
Check out [PEP-257](https://www.python.org/dev/peps/pep-0257/), which describes Pythonic docblocks. This is more akin to a C docblock